app/controllers/Api.scala

```diff
@@ -208,6 +208,13 @@ class Api (cc: ControllerComponents, prismDataStore: Prism, prismConfiguration:
       Api.singleItem(prismDataStore.reservationAgent, arn)
     }
 
+    def rdsList = Action.async { implicit request =>
+      Api.itemList(prismDataStore.rdsAgent, "rds")
```

Comment:
This should be a plural noun for consistency with others. Maybe `rds-instances` or something?